src/main/java/uk/gov/justice/digital/hmpps/csr/api/repository/SqlRepository.kt

```diff
@@ -135,17 +139,18 @@ class SqlRepository(private val jdbcTemplate: NamedParameterJdbcTemplate) {
     """.trimIndent()
 
     val GET_DETAIL_TEMPLATES = """
-            SELECT tk_modelItem.TASK_START AS startTime,
-                tk_modelItem.TASK_END AS endTime,
-                tk_modelItem.IS_FRAME_RELATIVE AS isRelative,
-                tk_type.NAME AS activity,
-                tk_model.NAME AS templateName
-                FROM tk_modelitem
-            JOIN tk_model ON tk_modelitem.TK_MODEL_ID = tk_model.TK_MODEL_ID
-            JOIN tk_type on tk_type.TK_TYPE_ID = tk_modelitem.TK_TYPE_ID
-            WHERE tk_model.NAME IN (:values)
-                AND tk_model.IS_DELETED = 0
-                AND tk_modelitem.taskstyle = 0
+         SELECT i.TASK_START AS startTime,
+                i.TASK_END AS endTime,
+                i.IS_FRAME_RELATIVE AS isRelative,
+                t.NAME AS activity,
+                m.NAME AS templateName
+         FROM tk_modelitem i
+            JOIN tk_model_info mi ON i.TK_MODEL_ID = mi.TK_MODEL_INFO_ID AND mi.IS_DELETED = 0
+            JOIN tk_model m ON mi.TK_MODEL_ID = m.TK_MODEL_ID AND m.IS_DELETED = 0
+            JOIN tk_type t on t.TK_TYPE_ID = i.TK_TYPE_ID
+            WHERE m.NAME IN (:values)
+              AND i.taskstyle = 0
```

Comment:
Need join on tk_model_info as i.TK_MODEL_ID is not a  TK_MODEL. TK_MODEL_ID, though most of these ids are the same hence went unnoticed for a long time!